java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -1029,4 +1031,345 @@ CompletableFuture<ClusterValue<Object>> fcallReadOnly(
      */
     CompletableFuture<ClusterValue<Map<GlideString, Map<GlideString, Object>>>> functionStatsBinary(
             Route route);
+
+    /**
+     * Invokes a Lua script.<br>
+     * This method simplifies the process of invoking scripts on the server by using an object that
+     * represents a Lua script. The script loading and execution will all be handled internally. If
+     * the script has not already been loaded, it will be loaded automatically using the <code>
+     * SCRIPT LOAD</code> command. After that, it will be invoked using the <code>EVALSHA </code>
+     * command.
```

Comment:
Add a line - routed to a random primary node (i hope so)

Reply:
resolving since i think this was intended for invokeScript(script)